.travis.yml

```diff
@@ -1,2 +1,8 @@
 script:
-- echo "hello world"
+- docker login --username devopscorndel --password $dockerhubpass
+- docker build --target production --tag devopscorndel/travisimage:latest .
+- docker push devopscorndel/travisimage:latest
+- docker login --username=rusab36@gmail.com --password=$herokutoken registry.heroku.com
+- docker pull devopscorndel/travisimage:latest
+- docker tag devopscorndel/travisimage:latest registry.heroku.com/devops-workshhop7/web
+- docker push registry.heroku.com/devops-workshhop7/web
```

Comment:
This updates heroku's image registry, but doesn't actually release the latest version, for that we need to actively tell Heroku to release the app with e.g. 
`heroku container:release -a devops-workshhop7 web`

Note that to do this we also need to login to the Heroku CLI - which is a separate step to logging into the Heroku registry. An easy way to do that is to name your heroku token precisely `HEROKU_API_KEY` - the Heroku CLI  will then pick that up by default